src/deploy/osp_deployer/director.py

```diff
@@ -1520,41 +1523,60 @@ def run_sanity_test(self):
             pass
 
     def configure_tempest(self):
-        logger.debug("configuring tempest")
+        logger.info("Configuring tempest")
         setts = self.settings
+        external_sub_cmd = ("source ~/" + setts.overcloud_name + "rc;" +
+                            "openstack subnet list  | grep external_sub " +
+                            "| awk '{print $6;}'")
+        external_sub_guid = self.run_tty(external_sub_cmd)[0].rstrip()
+
+        if not external_sub_guid:
+            logger.error("Could not find public network, please run the " +
+                         "sanity test to create the appropriate networks " +
+                         "and re-run this script with " +
+                         "--tempest_config_only flag.")
+            pass
```

Comment:
Do we want to about in this case instead of attempting to run tempest, since we know it won't work?